yarn-project/end-to-end/src/utils.ts

```diff
@@ -131,21 +157,21 @@ export async function setNextBlockTimestamp(rpcUrl: string, timestamp: number) {
  * @param abi - contracts to be deployed.
  * @returns The deployed contract instances.
  */
-export async function deployL2Contracts(aztecRpcServer: AztecRPCServer, abis: ContractAbi[]) {
-  const logger = getLogger();
-  const calls = await Promise.all(abis.map(abi => new ContractDeployer(abi, aztecRpcServer).deploy()));
-  for (const call of calls) await call.create();
-  const txs = await Promise.all(calls.map(c => c.send()));
-  expect(every(await Promise.all(txs.map(tx => tx.isMined(0, 0.1))))).toBeTruthy();
-  const receipts = await Promise.all(txs.map(tx => tx.getReceipt()));
-  const contracts = zipWith(
-    abis,
-    receipts,
-    (abi, receipt) => new Contract(receipt!.contractAddress!, abi!, aztecRpcServer),
-  );
-  contracts.forEach(c => logger(`L2 contract ${c.abi.name} deployed at ${c.address}`));
-  return contracts;
-}
+// export async function deployL2Contracts(aztecRpcServer: AztecRPCServer, abis: ContractAbi[]) {
```

Comment:
Should we delete this function, or restore it?

Reply:
Might as well delete if we are not using it